pkg/util/typeutil/schema.go

```diff
@@ -2464,6 +2464,10 @@ func ConcatStructFieldName(structName string, fieldName string) string {
 	return fmt.Sprintf("%s[%s]", structName, fieldName)
 }
 
+func ConcatStructFieldDefaultIndexName(structName string, fieldName string) string {
+	return fmt.Sprintf("%s_%s", structName, fieldName)
```

Comment:
what if we have a field that is in this format?

eg: we have a struct field `struct` and a subfield `a`, and we have another general field `struct_a`.